src/rcheevos/rc_client.c

```diff
@@ -219,7 +219,7 @@ static rc_clock_t rc_client_clock_get_now_millisecs(const rc_client_t* client)
     return 0;
 
   /* round nanoseconds to nearest millisecond and add to seconds */
-  return (rc_clock_t)(now.tv_sec * 1000 + (now.tv_nsec + 500000) / 1000000);
+  return ((rc_clock_t)now.tv_sec * 1000 + ((rc_clock_t)now.tv_nsec + 500000) / 1000000);
```

Comment:
Not sure I understand the +500000 addition here. There's not really any need for rounding here, in fact I think it's more consistent to round down always.

Reply:
Agreed. For a monotonic timer the rounding is irrelevant.